src/main/java/mods/railcraft/data/RailcraftLanguageProvider.java

```diff
@@ -80,7 +80,7 @@ private void blockTranslations() {
     this.addBlock(RailcraftBlocks.FORCE_TRACK_EMITTER, "Force Track Emitter");
 
     this.addBlockColorVariants(RailcraftBlocks.STRENGTHENED_GLASS, "Strengthened Glass");
-    this.addBlockColorVariants(RailcraftBlocks.POST, "Post");
+    this.addBlockColorVariants(RailcraftBlocks.POST, "Metal Post");
```

Comment:
I felt like having it as just "Post" because the other post variants were removed.

Reply:
Ok